temporalcli/commandsgen/commands.yml

```diff
@@ -917,34 +924,32 @@ commands:
   - name: temporal operator nexus
     summary: Commands for managing Nexus resources (EXPERIMENTAL)
     description: |
-      Nexus commands enable managing Nexus resources.
+      These commands manage Nexus resources.
 
       Nexus commands follow this syntax:
 
       ```
-      temporal operator nexus [command] [command] [command options]
+      temporal operator nexus [command] [subcommand] [options]
       ```
 
   - name: temporal operator nexus endpoint
     summary: Commands for managing Nexus Endpoints (EXPERIMENTAL)
     description: |
-      Endpoint commands enable managing Nexus Endpoints.
+      These commands manage Nexus Endpoints.
 
-      Endpoint commands follow this syntax:
+      Nexus Endpoint commands follow this syntax:
 
       ```
-      temporal operator nexus endpoint [command] [command options]
+      temporal operator nexus endpoint [command] [options]
       ```
 
   - name: temporal operator nexus endpoint create
-    summary: Create a new Nexus Endpoint (EXPERIMENTAL)
+    summary: Create a Nexus Endpoint (EXPERIMENTAL)
     description: |
-      Create a new Nexus Endpoint on the Server.
+      Create a Nexus Endpoint on the Server.
 
-      An endpoint name is used in workflow code to invoke Nexus operations.  The
-      endpoint target may either be a worker, in which case `--target-namespace` and
-      `--target-task-queue` must both be provided, or an external URL, in which case
-      `--target-url` must be provided.
+      A Nexus Endpoint name is used in Workflow code to invoke Nexus Operations.
+      The endpoint target may either be a Worker, in which case `--target-namespace` and `--target-task-queue` must both be provided, or an external URL, in which case `--target-url` must be provided.
```

Comment:
Should the endpoint here be uppercase since it still refers to a Nexus Endpoint?

Reply:
will fix